demo/src/main/java/com/orange/ods/demo/ui/components/buttons/ComponentButtonsContained.kt

```diff
@@ -16,100 +16,66 @@ import androidx.compose.ui.res.stringResource
 import com.orange.ods.compose.component.button.OdsButton
 import com.orange.ods.compose.theme.OdsDisplayAppearance
 import com.orange.ods.demo.R
-import com.orange.ods.demo.ui.utilities.composable.Subtitle
+import com.orange.ods.demo.ui.utilities.composable.Title
 
 @Composable
 fun ButtonsContained() {
-    Subtitle(R.string.component_buttons_contained_subtitle_primary, withHorizontalPadding = true)
-    OdsButton(
-        modifier = Modifier
-            .fullWidthButton(),
-        text = stringResource(R.string.component_state_enabled),
-        onClick = {},
-        hasPrimaryColor = true
-    )
-    OdsButton(
-        modifier = Modifier.fullWidthButton(false),
-        text = stringResource(R.string.component_state_disabled),
-        onClick = {},
-        enabled = false,
-        hasPrimaryColor = true
-    )
-
-    OdsButton(
-        modifier = Modifier.fullWidthButton(),
-        iconRes = R.drawable.ic_search,
-        text = stringResource(R.string.component_state_enabled),
-        onClick = {},
-        hasPrimaryColor = true
-    )
-    OdsButton(
-        modifier = Modifier.fullWidthButton(false),
-        iconRes = R.drawable.ic_search,
-        text = stringResource(R.string.component_state_disabled),
-        onClick = {},
-        enabled = false,
-        hasPrimaryColor = true
-    )
-
-    Subtitle(R.string.component_buttons_contained_subtitle_standard, withHorizontalPadding = true)
-    OdsButton(
-        modifier = Modifier.fullWidthButton(),
-        text = stringResource(R.string.component_state_enabled),
-        onClick = {},
-    )
-    OdsButton(
-        modifier = Modifier.fullWidthButton(false),
-        text = stringResource(R.string.component_state_disabled),
-        onClick = {},
-        enabled = false,
-    )
-
-    OdsButton(
-        modifier = Modifier.fullWidthButton(),
-        iconRes = R.drawable.ic_search,
-        text = stringResource(R.string.component_state_enabled),
-        onClick = {},
-    )
-    OdsButton(
-        modifier = Modifier.fullWidthButton(false),
-        iconRes = R.drawable.ic_search,
-        text = stringResource(R.string.component_state_disabled),
-        onClick = {},
-        enabled = false,
-    )
+    Title(R.string.component_buttons_contained_subtitle_primary, withHorizontalPadding = true)
+    EnabledDisabledContainedButtons(hasPrimaryColor = true, hasIcon = false)
+    EnabledDisabledContainedButtons(hasPrimaryColor = true, hasIcon = true)
+    LightSurface {
+        ContainedButtonsAppearanceForced(OdsDisplayAppearance.ON_LIGHT, hasPrimaryColor = true)
+    }
+    DarkSurface {
+        ContainedButtonsAppearanceForced(OdsDisplayAppearance.ON_DARK, hasPrimaryColor = true)
+    }
 
+    Title(R.string.component_buttons_contained_subtitle_standard, withHorizontalPadding = true)
```

Comment:
You can factorize this whole block (only `hasPrimaryColor` changes):

```kotlin
EnabledDisabledContainedButtons(hasPrimaryColor = true, hasIcon = false)
    EnabledDisabledContainedButtons(hasPrimaryColor = true, hasIcon = true)
    LightSurface {
        ContainedButtonsAppearanceForced(OdsDisplayAppearance.ON_LIGHT, hasPrimaryColor = true)
    }
    DarkSurface {
        ContainedButtonsAppearanceForced(OdsDisplayAppearance.ON_DARK, hasPrimaryColor = true)
    }
```

Same comment for text buttons.

Reply:
Fix is great :+1:
There is a typo, I renamed `TextButtonsEnabledDisable` to `TextButtonsEnabledDisabled` before merging.